tests/helix/send-to-helix-buildonhelixtests.targets

```
@@ -0,0 +1,75 @@
+<Project>
+
+  <PropertyGroup>
+    <BuildHelixWorkItemsDependsOn>$(BuildHelixWorkItemsDependsOn);BuildHelixWorkItemsForBuildOnHelixTests</BuildHelixWorkItemsDependsOn>
+    <NeedsWorkload>true</NeedsWorkload>
+    <NeedsPlaywright>true</NeedsPlaywright>
+
+    <WorkItemArchiveWildCard>$(TestArchiveTestsDirForBuildOnHelixTests)**/*.zip</WorkItemArchiveWildCard>
+
+    <_CleanupProcessesCommand Condition="'$(OS)' == 'Windows_NT'">powershell -ExecutionPolicy ByPass -NoProfile -command "&amp; get-ciminstance win32_process | where-object ExecutablePath -Match 'dotnet-latest|dcp.exe|dcpctrl.exe' | foreach-object { echo $_.ProcessId $_.ExecutablePath %3B stop-process -id $_.ProcessId -force -ErrorAction SilentlyContinue }"</_CleanupProcessesCommand>
+
+    <_CleanupProcessesCommand Condition="'$(OS)' != 'Windows_NT'">pgrep -lf "dotnet-latest|dcp.exe|dcpctrl.exe" | awk '{print %3B system("kill -9 "$1)}'</_CleanupProcessesCommand>
+
+    <_TestAssemblyRootDirEnvVar Condition="'$(OS)' != 'Windows_NT'">${HELIX_WORKITEM_ROOT}/test/</_TestAssemblyRootDirEnvVar>
+    <_TestAssemblyRootDirEnvVar Condition="'$(OS)' == 'Windows_NT'">%HELIX_WORKITEM_ROOT%\test\</_TestAssemblyRootDirEnvVar>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <HelixPreCommand Include="$(_EnvVarSetKeyword) SkipDashboardProjectReference=true" />
+    <HelixPreCommand Include="$(_EnvVarSetKeyword) TestsRunningOutsideOfRepo=true" />
+
+    <HelixPreCommand Condition="'$(OS)' == 'Windows_NT'" Include="set NUGET_PACKAGES=%HELIX_WORKITEM_ROOT%\nuget-cache\" />
+    <HelixPreCommand Condition="'$(OS)' != 'Windows_NT'" Include="export NUGET_PACKAGES=$HELIX_WORKITEM_ROOT/nuget-cache/" />
+
+    <HelixPreCommand Include="$(_CleanupProcessesCommand)" />
+
+    <HelixPostCommand Include="$(_CleanupProcessesCommand)" />
+  </ItemGroup>
+
+  <!-- `dotnet build` + `dotnet test` on helix -->
+  <Target Name="BuildHelixWorkItemsForBuildOnHelixTests">
+    <ItemGroup>
+      <_TestRunCommandArguments Remove="@(_TestRunCommandArguments)" />
+
+      <!-- Using `dotnet test` for the project directly here -->
+      <_TestRunCommandArguments Include="dotnet test" />
+      <_TestRunCommandArguments Include="-s .runsettings" />
+      <_TestRunCommandArguments Include="--results-directory $(_HelixLogsPath)" />
+      <_TestRunCommandArguments Include="--blame-hang" />
+      <_TestRunCommandArguments Include="--blame-hang-dump-type" />
+      <_TestRunCommandArguments Include="full" />
```

```suggestion
      <_TestRunCommandArguments Include="--blame-hang-dump-type full" />
```

Would it be more readable if these args were together? SAme for timeout and crash dump below.